homeassistant/helpers/entity.py

```diff
@@ -736,3 +746,53 @@ async def async_toggle(self, **kwargs: Any) -> None:
             await self.async_turn_off(**kwargs)
         else:
             await self.async_turn_on(**kwargs)
+
+
+class MeasurableUnitEntity(Entity):
+    """An abstract class for entities which state is a measurable unit."""
+
+    @property
+    def native_value(self) -> str | None:
```

Comment:
Native or raw?

Reply:
raw is not so good, I would expect that to be the raw binary value read out from the sensor, before conversion to a standard unit.